tests/Doctrine/Tests/ORM/Persisters/Exception/UnrecognizedFieldTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Persisters\Exception;
+
+use Doctrine\ORM\Persisters\Exception\UnrecognizedField;
+use Doctrine\Tests\Models\Taxi\Car;
+use PHPUnit\Framework\TestCase;
+
+class UnrecognizedFieldTest extends TestCase
+{
+    public function testByFullyQualifiedName(): void
+    {
+        static::expectException(UnrecognizedField::class);
+        static::expectExceptionMessage('Unrecognized field: Doctrine\Tests\Models\Taxi\Car::color');
```

Comment:
```suggestion
        static::expectExceptionMessage('Unrecognized field: Doctrine\Tests\Models\Taxi\Car::$color');
```

I would expect a `$` here since we're talking about a property and not a constant.